uiowa/uiowa
# Aggregator card style options

Resolves #2514 

Piggy backs on some tricks that card and headline implemented. Most of this should be similar for the other listings except for the image handling.

# Questions

- Should BG Color be passed to children? - _Yes, and not applied to the parent_
- Can other aggregator styles be cleaned up? No idea what the _row stuff does. - _Yes_
- Is my copy/pasta from card for the parent headline padding okay? - _No, since we want section controlling bg, this is not needed_
- **Not sure the caching solution for the feed image is smart.**
- Should the defaults match what is currently out in PROD (this would differ from the card default)? - _Setting to what is currently out there similar to how card set white as default_

# How to test

- Pull a v3 site, cim, blt frontend, configure Aggregator to pull in a feed. Place the RSS Feed block and test out the layout builder style options now available.
- New instances should match the styles here: https://sandbox.prod.drupal.uiowa.edu/aggregator-0

Example feed: http://blog.admissions.uiowa.edu/andre/feed/


<img width="952" alt="Screen Shot 2020-12-02 at 1 40 42 PM" src="https://user-images.githubusercontent.com/4663676/100922870-00584d80-34a4-11eb-9cbb-629afa7adaf4.png">
